packages/template-retail-react-app/app/components/item-variant/item-attributes.jsx

```diff
@@ -38,25 +40,68 @@ const ItemAttributes = ({includeQuantity, currency, ...props}) => {
         }
     )
     const promos = res?.data || []
-    // Create a mapping of variation values to their associated attributes. This allows us
-    // the render the readable names/labels rather than variation value IDs.
-    const variationValues = Object.keys(variant.variationValues || []).map((key) => {
-        const value = variant.variationValues[key]
-        const attr = variant.variationAttributes?.find((attr) => attr.id === key)
-        return {
-            id: key,
-            name: attr?.name || key,
-            value: attr.values.find((val) => val.value === value)?.name || value
+    const variationValues = getDisplayVariationValues(
+        variant?.variationAttributes,
+        variant?.variationValues
+    )
+
+    // get variant info for bundled products in cart page and order history page
+    const productBundleIds =
+        variant?.bundledProductItems?.map(({productId}) => productId).join(',') ?? ''
+    const {data: productBundleVariantData, isLoading: bundleVariantIsLoading} = useProducts(
+        {
+            parameters: {
+                ids: productBundleIds,
+                allImages: false
+            }
+        },
+        {
+            enabled: Boolean(variant?.type?.bundle && productBundleIds),
+            select: (result) => {
+                // formats response so we can easily display child quantity/variant selection
+                return result?.data?.map((item) => {
+                    const quantity = variant?.bundledProductItems.find(
+                        (childProduct) => childProduct.productId === item.id
+                    )?.quantity
+                    return {
+                        ...item,
+                        quantity,
+                        variationValues: getDisplayVariationValues(
+                            item?.variationAttributes,
+                            item?.variationValues
+                        )
+                    }
+                })
+            }
+        }
+    )
+
+    // get bundle product data for wishlist page
+    const {data: productBundleData, isLoading: productBundleIsLoading} = useProduct(
+        {
+            parameters: {
+                id: variant?.id,
+                allImages: false
+            }
+        },
+        {
+            enabled: Boolean(variant?.type?.bundle && !productBundleIds)
         }
-    })
+    )
 
     return (
         <Stack spacing={1.5} flex={1} {...props}>
-            {variationValues?.map((variationValue) => (
-                <Text lineHeight={1} color="gray.700" fontSize="sm" key={variationValue.id}>
-                    {variationValue.name}: {variationValue.value}
-                </Text>
-            ))}
+            {variationValues &&
+                Object.keys(variationValues).map((key) => (
```

Comment:
Is this a breaking change? Updating the implementation of the rendering since the data is structured differently? The UI looks the same